example.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from itertools import pairwise
+from pathlib import Path
+from typing import Any, Iterable, Iterator, Mapping, Tuple
+
+from telcell.data.models import Track
+from telcell.data.parsers import parse_measurements_csv
+from telcell.data.utils import extract_intervals, split_track_by_interval
+from telcell.models import DummyModel
+from telcell.pipeline import run_pipeline
+
+
+def dummy_cruncher(tracks: Iterable[Track]) \
+        -> Iterator[Tuple[Track, Track, Mapping[str, Any]]]:
```

Comment:
Dit lijkt me een logische plek om ook de label terug te geven `Tuple[Track, Track, Mapping[str, Any], bool]`

Reply:
Wil je het label apart teruggeven als het zo makkelijk is om `is_colocated()` aan te roepen? Dit maakt namelijk de return type wel nog ingewikkelder (en hij is al vrij ingewikkeld), en dit is juist het stuk code wat mensen zelf moeten implementeren dus waar ze ook direct mee te maken krijgen. Vandaar dat ik deze return type (en signature algemeen) het liefst zo simpel mogelijk zou houden.